pkg/syncer/k8scloudoperator/k8scloudoperator.go

```diff
@@ -305,6 +307,15 @@ func (k8sCloudOperator *k8sCloudOperator) PlacePersistenceVolumeClaim(ctx contex
 		return out, nil
 	}
 
+	// Abort placement if the storage class has Immediate volume binding and nodeAffinity PVC annotation is not specified
+	if *sc.VolumeBindingMode != storagev1.VolumeBindingWaitForFirstConsumer {
+		if _, ok := pvc.ObjectMeta.Annotations[nodeAffinityAnnotationKey]; !ok {
+			log.Debugf("Aborting placement for PVC[%s] as neither volume binding of the storage class is [%s]"+
```

Comment:
nit: Add one space between the concatenated messages
`[%s]"+ ==> [%s] "+`
or
 `"nor ==> " nor`